### PR TITLE
Convert Venmo demo to Swift

### PR DESCRIPTION
**Pull request by scannillo**
### Summary of changes

We recently got an inbound (Issue #1085) regarding the Swift async-await `VenmoClient.tokenize()` function. To more easily be able to test async await methods, this PR converts our Venmo feature demo view controller to Swift.

### Changes

- Convert Venmo demo ViewController to Swift
- Add second button to Venmo Demo for ECD params
- Update UI tests

### Future

In a follow up convo we can discuss migrating all feature demos to Swift, as well as how we want to test async-await methods.

### Checklist

- ~Added a changelog entry~

### Authors
@scannillo @jaxdesmarais

### CODE REVIEW

File: Demo/Application/Features/Venmo - Custom Button/BraintreeDemoVenmoViewController.swift [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import UIKit
+import BraintreeVenmo
+
+class BraintreeDemoVenmoViewController: BraintreeDemoPaymentButtonBaseViewController {
+    
+    var venmoClient: BTVenmoClient!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        venmoClient = BTVenmoClient(apiClient: self.apiClient)
```
Comment (jaxdesmarais) [inline Demo/Application/Features/Venmo - Custom Button/BraintreeDemoVenmoViewController.swift:10]:
```suggestion
        venmoClient = BTVenmoClient(apiClient: apiClient)
```

File: Demo/Application/Features/Venmo - Custom Button/BraintreeDemoVenmoViewController.swift [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        self.progressBlock("Tapped Venmo - initiating Venmo auth")
+        
+        let venmoRequest = BTVenmoRequest(paymentMethodUsage: .multiUse)
+        venmoRequest.vault = true
+        venmoRequest.collectCustomerBillingAddress = true
+        venmoRequest.collectCustomerShippingAddress = true
+        venmoRequest.totalAmount = "30.00"
+        venmoRequest.taxAmount = "1.10"
+        venmoRequest.discountAmount = "1.10"
+        venmoRequest.shippingAmount = "0.00"
```
Comment (jaxdesmarais) [inline Demo/Application/Features/Venmo - Custom Button/BraintreeDemoVenmoViewController.swift:32]: I wonder if we should do something like we have in the NXO ViewController and have a separate button for the standard vs the enriched customer data flow. Would be an easy way to test both flows and could have prevented the dictionary bug if we had both buttons on the same screen!

**Review by jaxdesmarais — APPROVED** [review]

File: Demo/Application/Features/Venmo - Custom Button/BraintreeDemoVenmoViewController.swift (hunk 2 above)
Comment (scannillo) [reply Demo/Application/Features/Venmo - Custom Button/BraintreeDemoVenmoViewController.swift:67]: Done!

File: Demo/Demo.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 52;
+	objectVersion = 54;
```
Comment (scannillo) [inline Demo/Demo.xcodeproj/project.pbxproj:6]:
```suggestion
	objectVersion = 52;
```

**Pull request merged**